recipes/shellcheck/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "shellcheck" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "348a3f7892c1f28a44f188c00ac82f1b3bf899d9f81d14ddb0e306db26c937bb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/koalaman/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - stack
+
+  run:
+    - gmp 6.1.*
```

Comment:
Would relax the pinning as we add this automatically. Also please add a `host` section with `gmp` as well.

Reply:
Ok

Reply:
For the host section I am not sure. The dependency is not coming from the package itself. It is coming via the compiler (stack). That is like {{native}}gcc-libs, which is only added to run in many recipes, but not to host.